### DISCUSSION TITLE
v8.0.0 broke my install

### DISCUSSION THREAD

**Discussion by YeapGuy** [nextcloud/all-in-one#4441] [Questions] [answered]
After upgrading to v8.0.0, I get "No files", even though the files are there, and it was working perfectly before upgrading.
![image](https://github.com/nextcloud/all-in-one/assets/43749899/5087139f-aa81-47af-906e-cff50ecf4644)
Two things:
1. What happened?? Can I help debug this? There's nothing related in the logs...
2. How can I roll back? I restored all the docker volumes and returned my images to previous ones based on sha256 digest (I'm managing the containers myself, "manual-install"). Even then, it's in a broken state. Do I need to restore the datadir as well?

**Comment by johnnyq** [comment]
Hey @YeapGuy Do you have external File Plugin Enabled?
Disable External Files Plugin and let me know if this worked

**Reply by YeapGuy** [reply]
I do, and I need it. Is it broken in v28?

**Reply by johnnyq** [reply]
It seems to be some kind of bug but a workaround that worked for me was disconnect each external share then renabled them and it worked. My external shares are FTP servers to a webhost as our copiers send scans to that webhost and nextcloud pulls them down via ftp external files plugins

**Comment by szaimen** [comment, marked as answer]
Hi, you should be able to restore the working instance by restoring the last backup before upgrading to v8.0.0 via the AIO interface
